src/main/fc/core.h

```diff
@@ -47,6 +47,18 @@ typedef enum {
     LAUNCH_CONTROL_MODE_COUNT // must be the last element
 } launchControlMode_e;
 
+typedef enum FlightLogDisarmReason {
```

Comment:
Enum types in Betaflight are normally defined without a name, and with a lowercase type ending in `_e`, see the enum types defined right above.

Also, adding comments indicating the meaning of these (at least for the less obvious ones like `DISARM_REASON_SERIAL_IO` will make this easier to grasp for others.

Reply:
I think the enum was moved from it's original location in the blackbox code where the enums are not currently defined using the `_e` convention.  Will fix, but likely you may want a to fix the blackbox enums at some point too.

I'd rather not add comments, I'd prefer well named enum values that require no comments, do you have suggestions for alternate names for ones you're not happy with?